server/bot_messages.go

```diff
@@ -76,9 +76,8 @@ func (p *Plugin) SendWelcomeMessageWithNotificationAction(userID string) error {
 
 func (p *Plugin) makeWelcomeMessageWithNotificationActionPost() *model.Post {
 	msg := []string{
-		"**Notifications from chats and group chats**",
-		"When you enable this feature, you'll be notified here in Mattermost whenever you're away from Microsoft Teams and receive a message from a chat or group chat.",
-		fmt.Sprintf("![enable notifications picture](%s/static/enable_notifications.gif)", p.GetRelativeURL()),
+		"**Get notified for MS Teams Chats**",
```

Comment:
Why is this necessary if we are defaulting the notifications to enabled? Can we just remove this and only fire the one message with GIF from [here](https://github.com/mattermost/mattermost-plugin-msteams/pull/740/files#diff-3fb3d4184590bc6f747661f3062676031e837b0dd5f2b18640f439be7a2235f6R961) once their account is connected?